packages/engine/Specs/DataSources/CallbackPositionPropertySpec.js

```diff
@@ -7,7 +7,10 @@ import {
 } from "../../index.js";
 
 describe("DataSources/CallbackPositionProperty", function () {
-  const time = JulianDate.now();
+  let time;
+  beforeEach(function () {
+    time = JulianDate.now();
+  });
```

Comment:
This change is unrelated, but I noticed it was interfering with other specs on a test failure.